backend/typescript/services/implementations/camperService.ts

```diff
@@ -602,6 +605,318 @@ class CamperService implements ICamperService {
     return updatedCamperDTOs;
   }
 
+  async moveCampersToCampSession(
+    camperIds: string[],
+    campSessionId: string,
+  ): Promise<CamperDTO[]> {
+    const session = await mongoose.startSession();
+    await session.startTransaction();
```

Comment:
is this await necessary? I got a warning saying this is not necessary and the function documentation doesn't mention anything about Promises